README.md

```diff
@@ -1732,6 +1732,7 @@ Software for easy installation, management and configuration of self-hosted serv
 - [Sandstorm](https://sandstorm.io/) - Personal server for running self-hosted apps easily and securely. ([Demo](https://demo.sandstorm.io/), [Source Code](https://github.com/sandstorm-io/sandstorm)) `Apache-2.0` `C++/Other`
 - [sovereign](https://github.com/sovereign/sovereign) - Set of Ansible playbooks to build and maintain your own private cloud: email, calendar, contacts, file sync, IRC bouncer, VPN, and more. `GPL-3.0` `YAML/Other`
 - [Syncloud](https://syncloud.org/) - Your own online file storage, social network or email server. ([Source Code](https://github.com/syncloud/platform)) `GPL-3.0` `Python/Other`
+- [Tipi](https://github.com/meienberger/runtipi) - Homeserver manager. One command setup, one click installs for your favorites self-hosted apps. `GPL-3.0` `Docker/Shell`
```

Comment:
You mention Docker can be used within the language tag however this is not present within the installation section on the github, is there another place for installation docs?

There is a demo associated with this project, maybe this could also be added?

Reply:
It is not mentioned in the docs because the shell scripts installs docker for the user if not present. So at the end only shell is required. Do I need to remove it ?

Reply:
If I link the demo here, I would also need to provide the `username` and `password` and I didn't see any other entry on this list doing so

Reply:
I'd go ahead and remove it. Leaving docker there suggests that theres a command to run and such.

Nodiscc may have different feedback so you may have to re-add it but currently Id say remove it.

Reply:
> If I link the demo here, I would also need to provide the `username` and `password` and I didn't see any other entry on this list doing so

Other demo links on this repo do this, having the demo link atleast suggests that there is a demo available.